user_manual/files/projects.rst

```diff
@@ -2,12 +2,15 @@
 Projects
 ========
 
+.. deprecated:: 25
+    This feature was replaced by the shipped `related resources <https://apps.nextcloud.com/apps/related_resources>`_ app.
```

Comment:
https://github.com/nextcloud/server/blob/master/core/shipped.json#L36

Shipped by now, so the link to a no longer updated appstore entry might be confusing as well.

Reply:
Removed the link.